packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -38,111 +22,117 @@ Chains not supported yet due fireblocks or zerodev doesn't supported
     CARDANO
 */
 
-export function getFireblocksAssetId(chain: BlockchainIncludingTestnet) {
-    const assetId = new Map([
+export function getFireblocksAssetId(chain: EVMBlockchainIncludingTestnet) {
+    const assetId = new Map<EVMBlockchainIncludingTestnet, string>([
         ["ethereum", "ETH"],
+        ["ethereum-goerli", "ETH_TEST3"],
+        ["ethereum-sepolia", "ETH_TEST5"],
         ["polygon", "MATIC_POLYGON"],
+        ["polygon-mumbai", "MATIC_POLYGON_MUMBAI"],
         ["bsc", "BNB_BSC"],
         ["optimism", "ETH-OPT"],
+        ["optimism-sepolia", "ETH-OPT_KOV"],
+        ["optimism-goerli", "ETH-OPT_KOV"],
         ["arbitrum", "ETH-AETH"],
-        ["goerli", "ETH_TEST3"],
-        ["ethereum-sepolia", "ETH_TEST5"],
-        ["mumbai", "MATIC_POLYGON_MUMBAI"],
+        ["arbitrum-sepolia", "ETH-AETH_RIN"],
+        ["base-sepolia", "ETH_TEST3"],
+        ["base-goerli", "ETH_TEST3"],
+        ["zora", "ETH"],
+        ["zora-sepolia", "ETH_TEST3"],
+        ["zora-goerli", "ETH_TEST3"],
         ["zkatana", "ETH_ZKEVM_TEST"],
+        ["bsc-testnet", ""],
+        ["base", ""],
     ]).get(chain)!;
 
-    if (assetId == null) {
-        throw new Error(`Url not found for chain ${chain}`);
+    if (assetId == null || assetId === "") {
```

Comment:
what do you think about this:
```
const assetId = new Map<EVMBlockchainIncludingTestnet, string | null>([
```
so that we can do 
```
["bsc-testnet", null],
```
and then leave this as:
```
if (assetId == null)
```